src/style.css

```
@@ -0,0 +1,121 @@
+body,
+h1,
+p,
+img {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  max-height: 404px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  max-width: 160px;
+  margin: 32px auto 40px;
+}
+
+.text {
+  padding: 0 16px 16px;
+}
+
+.text__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 2px;
+}
+
+.text__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.review,
+.cost {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #060b35;
+}
+
+.review {
+  padding: 16px 0 24px;
```

```suggestion
  margin-bottom: 24px;
```

Use margin-bottom for position